### DISCUSSION TITLE
Welcome to StayPut Discussions!

### DISCUSSION THREAD

**Discussion by pmdevita** [pmdevita/StayPut#19] [Announcements]
Forgot to do this before, figure I'd open this up in case people are having difficulty configuring or setting up the plugin.  If you need help, aren't sure if you found a bug or want to discuss a new feature, post here! If the plugin isn't working right, it's most helpful if you can post your server log with the plugin in debug mode.

**Comment by SirSolis** [comment]
its not saving my locations at all whenever i teleport i'm using mysql workbench
issue: i spawn in my spawn world go thru my multiverse portal and it spawns me at the default spawnpoint i move far away then head back to the spawn world  and when i use the teleportal it takes me back to the spawn point

**Reply by pmdevita** [reply]
Turn on debug mode and post the log from when you teleport
